packages/hoverable/src/with-hover.spec.tsx

```diff
@@ -0,0 +1,110 @@
+// tslint:disable-next-line:no-import-side-effect
+import '@testing-library/jest-native/extend-expect';
+import { cleanup, fireEvent, NativeTestEvent, render } from '@testing-library/react-native';
+import React, { FC } from 'react';
+import { StyleProp, Text, TextStyle, View } from 'react-native';
+
+import { withHover } from './with-hover.hoc';
+
+interface Props {
+    style?: StyleProp<TextStyle>;
+    testID?: string;
+}
+
+enum SelectorEnum {
+    BaseComponent = 'BaseComponent',
+}
+
+const Component: FC<Props> = ({ children, ...props }) => <View {...props}>{children}</View>;
+const HoverableViewComponent = withHover(Component);
+const hoverStyle = { backgroundColor: 'black' };
+const style = { width: 100, backgroundColor: 'white' };
+
+describe('withHover HOC', () => {
+    afterEach(cleanup);
+    describe('Basic behavior of hoverable component', () => {
+        it('Should merge hoverStyle with current style to component styles onMouseEnter', () => {
+            const { getByTestId } = render(
+                <HoverableViewComponent testID={SelectorEnum.BaseComponent} style={style} hoverStyle={hoverStyle} />
+            );
+            const element = getByTestId(SelectorEnum.BaseComponent);
+            expect(element).toHaveStyle(style);
```

Comment:
toHaveStyle is inclusive, searching for any parts of the style, better to search fot .not.toHaveStyle https://github.com/testing-library/jest-native

Reply:
yep, this will be more explicit)